types/react/canary.d.ts

```diff
@@ -119,6 +121,21 @@ declare module "." {
         onTransitionStartCapture?: TransitionEventHandler<T> | undefined;
     }
 
+    type ToggleEventHandler<T = Element> = EventHandler<ToggleEvent<T>>;
+
+    interface HTMLAttributes<T> {
+        popover?: "" | "auto" | "manual" | undefined;
```

Comment:
The type here is missing `"hint"` as per https://developer.mozilla.org/en-US/docs/Web/API/Popover_API/Using#creating_the_tooltips_with_popoverhint